src/uikit/Input.js

```diff
@@ -2,4 +2,6 @@ import styled from 'react-emotion';
 
 export default styled('input')`
   ${({ theme }) => theme.input};
+  ${({ italic }) => (italic ? `font-style: italic` : ``)};
+  ${({ flex }) => (flex ? `flex: ${flex}` : '')};
```

Comment:
should we just introduce styled-system? https://github.com/jxnblk/styled-system/blob/master/docs/table.md

actually, couldn't find font-style, but it seems to do almost everything else

Reply:
yeah I'll do that